kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -71,12 +71,12 @@ public BigQueryRecordConverter(boolean shouldConvertDoubleSpecial) {
    *                           consist of both a name and a value.
    * @return The result BigQuery row content.
```

Comment:
nit: update doc regarding `convertKey `

Reply:
Removed the boolean `convertKey` and updated the doc as well.

Reply:
👍